conference.js

```diff
@@ -796,44 +785,55 @@ export default {
 
             APP.store.dispatch(makePrecallTest(this._getConferenceOptions()));
 
-            const { tryCreateLocalTracks, errors } = this.createInitialLocalTracks(initialOptions);
-            const tracks = await tryCreateLocalTracks;
+            return APP.store.dispatch(executeTrackOperation(TrackOperationType.AudioVideo, () => {
+                const { tryCreateLocalTracks, errors } = this.createInitialLocalTracks(initialOptions);
 
-            // Initialize device list a second time to ensure device labels
-            // get populated in case of an initial gUM acceptance; otherwise
-            // they may remain as empty strings.
-            this._initDeviceList(true);
+                return tryCreateLocalTracks.then(tracks => {
+                    // Initialize device list a second time to ensure device labels
+                    // get populated in case of an initial gUM acceptance; otherwise
+                    // they may remain as empty strings.
+                    this._initDeviceList(true);
 
-            if (isPrejoinPageVisible(APP.store.getState())) {
-                return APP.store.dispatch(initPrejoin(tracks, errors));
-            }
+                    if (isPrejoinPageVisible(APP.store.getState())) {
+                        return APP.store.dispatch(initPrejoin(tracks, errors));
+                    }
 
-            logger.debug('Prejoin screen no longer displayed at the time when tracks were created');
+                    logger.debug('Prejoin screen no longer displayed at the time when tracks were created');
 
-            this._displayErrorsForCreateInitialLocalTracks(errors);
+                    this._displayErrorsForCreateInitialLocalTracks(errors);
 
-            let localTracks = handleStartAudioMuted(initialOptions, tracks);
+                    let localTracks = handleStartAudioMuted(initialOptions, tracks);
 
-            // in case where gum is slow and resolves after the startAudio/VideoMuted coming from jicofo, we can be
-            // join unmuted even though jicofo had instruct us to mute, so let's respect that before passing the tracks
-            if (!browser.isWebKitBased()) {
-                if (room?.isStartAudioMuted()) {
-                    localTracks = localTracks.filter(track => track.getType() !== MEDIA_TYPE.AUDIO);
-                }
-            }
+                    // in case where gum is slow and resolves after the startAudio/VideoMuted coming from jicofo, we
```

Comment:
So I don't think we should ignore this since if I understand correctly those tracks are going to be added after we have received the start muted from jicofo and therefore the newly added tracks won't be removed by LJM.

Otherwise I don't find moving the code for the mute in  `handleStartAudioMuted` related to the PR. But if you insist on it I'll move it. Do you want me to?